src/sentry/static/sentry/app/views/settings/project/projectProcessingIssues.tsx

```diff
@@ -199,50 +211,56 @@ class ProjectProcessingIssues extends React.Component {
     });
   };
 
-  renderDebugTable = () => {
-    let body;
-    if (this.state.loading) {
+  renderDebugTable() {
+    let body: React.ReactNode;
+    const {loading, error, processingIssues} = this.state;
+    if (loading) {
       body = this.renderLoading();
-    } else if (this.state.error) {
+    } else if (error) {
       body = <LoadingError onRetry={this.fetchData} />;
     } else if (
-      this.state.processingIssues.hasIssues ||
-      this.state.processingIssues.resolveableIssues ||
-      this.state.processingIssues.issuesProcessing
+      processingIssues &&
+      (processingIssues.hasIssues ||
+        processingIssues.resolveableIssues ||
```

Comment:
maybe we should use the "`defined`" function here? because as it is a number, it can happen that we will get a 
`0` value here

Reply:
If all the counts are 0, then there are no processing issues, and we can show the empty state.